en/docs/develop-components/sharing-and-reusing/create-a-connection.md

```diff
@@ -1,32 +1,67 @@
+    7. Click **Create**.
+   
+    This creates the connection and displays its details for each environment, along with an inline guide on how to use the connection in your component. 
 
-1. Follow steps 1–4 from the previous section.
-2. Expand the **Environment to Endpoint** section and verify the service endpoints assigned to the Choreo environments.
-3. Click **Create**.
+    <h2>Create a connection to an external service</h2>
 
-For step-by-step instructions on using a connection in your service, see [Use a Connection in Your Service](./use-a-connection-in-your-service.md).
+    Follow these steps to create a connection to an external service:
 
-For step-by-step instructions on using a connection in your web application, see [Use a Connection in Your Web Application](./use-a-connection-in-your-web-application.md).
+    1. Follow steps 1–5 from the previous section.
+    2. Expand the **Environment to Endpoint** section and verify the service endpoints assigned to the Choreo environments.
+    3. Click **Create**.
+
+      For step-by-step instructions on using a connection in your service, see [Use a Connection in Your Service](./use-a-connection-in-your-service.md).
+
+      For step-by-step instructions on using a connection in your web application, see [Use a Connection in Your Web Application](./use-a-connection-in-your-web-application.md).
+
+
+=== "Create a connection to a database"
+
+    **Prerequisites**:
+
+     - Create a Choreo-managed database. For details, see [Choreo-Managed Databases and Caches](../../manage-databases-and-caches/choreo-managed-databases-and-caches.md).
+     - Add the database to the Marketplace. For details, see 
+
+    Follow these steps to create a connection to a Choreo-managed database:
+
+    1. In the Choreo Console, go to the top navigation menu and set the visibility level as [project](../../choreo-concepts/connections.md#project-connections) or [component](../../choreo-concepts/connections.md#component-connections) as follows: 
+
+        - **Project Connection**: Select an organization and a project in that organization. 
+        - **Component Connection**: Select an organization, a project in that organization, and a component in the selected project. 
+
+    2. In the left navigation menu, click **Dependencies**  and then **Connections**. This page lists all the existing connections.
+    3. Click **+Create**. This opens the Marketplace view where you can browse and search for services or databases.
+    4. Click the **Databases** tab. You can search and apply filters to efficiently find a database.
+    5. Click on the database you want to connect to. 
+    6. To create the connection, follow these steps:
+       1. Enter a name and description.
+       2. Under **Environment Configuration**, select credentials for each environment.
```

Comment:
Shall we add  this part as well

By default, the selected database will be applied to all environments. If you want use different databases across environments, select the appropriate database and corresponding credentials for each environment.

Reply:
Done